tin/apps/assignments/tests.py

```diff
@@ -16,20 +15,20 @@ def test_create_folder(client, course) -> None:
 
 
 @teacher
-@pytest.mark.parametrize("is_quiz", (-1, 0, 1, 2))
```

Comment:
This seems like a good opportunity to add tests for methods like `quiz_ended_for_student`, `quiz_locked_for_student` and `quiz_issues_for_student`. Maybe in a follow-up PR?

Reply:
Yep - the plan for this PR is just a one-to-one model migration.